### DISCUSSION TITLE
Add fixVersions support to jira issue list

### DISCUSSION THREAD

**Discussion by rustyx** [ankitpokhrel/jira-cli#673] [Ideas]
My JIRA ticket does contain "fixVersions" information, but I couldn't fetch it using jira-cli.

```json
    "fixVersions": [
      {
        "self": "https://example.atlassian.net/rest/api/3/version/18524",
        "id": "18524",
        "description": "",
        "name": "r2023.04.00",
        "archived": false,
        "released": false,
        "releaseDate": "2023-11-17"
      }
    ],
```

How to get the version name(s) (e.g. "r2023.04.00" above) using jira-cli?

**Comment by ankitpokhrel** [comment]
Hi @rustyx, I think you can use raw JQL for this?

```
 -q, --jql string              Run a raw JQL query in a given project context
```

**Comment by rustyx** [comment]
> I think you can use raw JQL for this?

I don't need to query on `fixVersions`, I need to fetch `fixVersions` for my jira tickets. I don't see how JQL can help with that?

**Comment by rustyx** [comment]
I think it's currently impossible to display `fixVersions` since it's not listed in [fields.do](https://github.com/ankitpokhrel/jira-cli/blob/main/internal/view/fields.go)?
